src/main/java/org/apache/commons/dbcp2/BasicDataSource.java

```diff
@@ -1794,7 +1794,7 @@ public void setConnectionFactoryClassName(final String connectionFactoryClassNam
      */
     public void setConnectionInitSqls(final Collection<String> connectionInitSqls) {
         final List<String> collect = Utils.isEmpty(connectionInitSqls) ? null
-                : connectionInitSqls.stream().filter(s -> !isEmpty(s)).collect(Collectors.toList());
+            : connectionInitSqls.stream().filter(s -> !isEmpty(s)).collect(Collectors.toList());

```

Comment:
Please don't change the formatting, it makes PR noisier and takes longer to review.